vars/standardReleasePipelineWithGenericTrigger.groovy

```diff
@@ -63,18 +63,18 @@ void call(Map args = [:], Closure body) {
         post {
             success {
                 script {
-                    if (args.publishRelease && release_url!= null){
-                        withCredentials([string(credentialsId: 'jenkins-github-bot-token', variable: 'GIT_TOKEN')]){
-                            sh "curl -X PATCH -H 'Accept: application/vnd.github+json' -H 'Authorization: Bearer ${GIT_TOKEN}' ${release_url} -d '{\"tag_name\":\"${tag}\",\"draft\":false,\"prerelease\":false}'"
-                            } 
-                    } 
+                    if (args.publishRelease && release_url != null) {
+                        withCredentials([usernamePassword(credentialsId: "${GITHUB_BOT_TOKEN_NAME}", usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_TOKEN')]) {
```

Comment:
We can continue to use `jenkins-github-bot-token` right, since it wont be changing much. @gaiksaya

Reply:
Agreed! Made the changes.